runtimes/common/src/dip/deposit/mod.rs

```diff
@@ -16,22 +16,34 @@
 
 // If you feel like getting in touch with us, you can do so at info@botlabs.org
 
-use crate::{DidIdentifier, Runtime};
 use frame_support::traits::Get;
 use pallet_deposit_storage::{
 	traits::DepositStorageHooks, DepositEntryOf, DepositKeyOf, FixedDepositCollectorViaDepositsPallet,
 };
 use pallet_dip_provider::IdentityCommitmentVersion;
 use parity_scale_codec::{Decode, Encode, MaxEncodedLen};
-use runtime_common::{constants::dip_provider::COMMITMENT_DEPOSIT, AccountId};
 use scale_info::TypeInfo;
 use sp_core::{ConstU128, RuntimeDebug};
 
+use crate::{constants::dip_provider::COMMITMENT_DEPOSIT, AccountId, DidIdentifier};
+
+#[cfg(test)]
+mod mock;
+#[cfg(test)]
+mod tests;
+
 #[derive(Encode, Decode, MaxEncodedLen, TypeInfo, Clone, PartialEq, Eq, RuntimeDebug)]
 pub enum DepositNamespace {
 	DipProvider,
 }
 
+#[cfg(feature = "runtime-benchmarks")]
+impl Default for DepositNamespace {
+	fn default() -> Self {
+		Self::DipProvider
+	}
+}
+
```

Comment:
Mhh but it would not really make sense, would it? Why would you have a default value for a namespace, when this is supposed to be used "with caution" when dealing with funds?

Reply:
Yes okay. I just saw that the Enum has only one entry. This might change in future.